templates/common/copilot-plugin-existing-api/.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Preview in Copilot (Edge)",
+            "type": "msedge",
+            "request": "launch",
+            "url": "https://teams.microsoft.com/l/app/${{TEAMS_APP_ID}}?installAppPackage=true&webjoin=true&${account-hint}",
```

Comment:
No need to open the app. Just `https://teams.microsoft.com?${account-hint}` is fine.

Reply:
updated.